Userland/Tests/CMakeLists.txt

```diff
@@ -3,3 +3,4 @@ add_subdirectory(LibC)
 add_subdirectory(LibGfx)
 add_subdirectory(LibM)
 add_subdirectory(UserspaceEmulator)
+add_subdirectory(AK)
```

Comment:
Thanks for fixing this so fast! :D

Could you sort this alphabetically? I believe "AK" goes to the top.